packages/node_modules/nav-frontend-stegindikator/src/stegindikator.tsx

```diff
@@ -0,0 +1,237 @@
+import * as React from 'react';
+import * as cn from 'classnames';
+
+import StegindikatorSteg, { StegindikatorStegProps } from './stegindikator-steg';
+import { omit } from 'nav-frontend-js-utils';
+
+import 'nav-frontend-stegindikator-style';
+
+const cls = (state) => cn('stegindikator', {
+    'stegindikator--kompakt': state.kompakt
+});
+
+export interface StegindikatorProps {
+    steg: StegindikatorStegProps[];
+    visLabel: boolean;
+    kompakt: boolean;
+    aktivtSteg?: number;
+    onChange?: (index: number) => void;
+    onBeforeChange?: (index: number) => boolean;
+    autoResponsiv: boolean;
+}
+
+export interface StegindikatorState {
+    aktivtSteg: number;
```

Comment:
Har ikke enkelt steget en `aktiv` prop? Isåfall trengs det vel ikke state-håndtering på dette?
`visLabel`  og `kompakt` kommer også inn som props, så de trenger vel heller ikke å ligge i state, eller er det noe ikke ser?

Reply:
Ettersom vi gikk for `onChange` fremfor `onClick` så må komponenten holde track på hva som forandres. `visLabel` og `kompakt` ligger på state pga. `autoResponsiv` (se f.eks. `adjustSize` metoden).